data_juicer/utils/lazy_loader.py

```diff
@@ -0,0 +1,44 @@
+# Code copied from https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/util/lazy_loader.py # noqa: E501
```

Comment:
The content in the link is inconsistent with this file; please check. 

Alternatively, it can be found at: https://github.com/tensorflow/tensorflow/blob/d5803aa0ad910afbeaff8c0a9c4316d220c7903a/tensorflow/python/keras/utils/generic_utils.py#L1160-L1181.